astropy/constants/constant.py

```diff
@@ -60,6 +60,7 @@ def wrapper(self, *args, **kwargs):
 
         # The wrapper applies to so many of the __ methods that it's easier to
         # just exclude the ones it doesn't apply to
+        # FIXME: replace __nonzero__ with __bool__ ?
```

Comment:
Not sure who to ping here (@mhvk maybe?), but there is a `__nonzero__` use that should probably be replaced with `__bool__` ?

Reply:
Yes, that can now be replaced by `__bool__` (which is python3, `__nonzero__` is python2).

Reply:
👍